syntax = 'proto3';

message Features {
    float MSSubClass     = 1 ;
    float LotArea        = 2 ;
    float YearBuilt      = 3 ;
    float BedroomAbvGr   = 4 ;
    float TotRmsAbvGrd   = 5 ;
}

message Empty {
}

service get_next_row {
    rpc get_next_row(Empty) returns(Features);
}